tutor/templates/apps/openedx/settings/partials/common_all.py

```diff
@@ -157,7 +157,6 @@
 warnings.filterwarnings("ignore", category=DeprecationWarning, module="wiki.plugins.links.wiki_plugin")
 warnings.filterwarnings("ignore", category=DeprecationWarning, module="boto.plugin")
 warnings.filterwarnings("ignore", category=DeprecationWarning, module="botocore.vendored.requests.packages.urllib3._collections")
-warnings.filterwarnings("ignore", category=DeprecationWarning, module="pkg_resources")
```

Comment:
This warning is not related to tutor, but to edx-platform. We should preserve it.

Reply:
Alright, we'll add it back.